wiki/Tournaments/NPC/2/en.md

```diff
@@ -0,0 +1,220 @@
+---
+
+tags:
+  - NPC
+  - NPC2
+---
+
+# Non-Professional Cup 2: Summer 2020
+
+![NPC2 Logo](images/banner-qmv2.png)
+
+The **Non-Professional Cup 2** (***NPC2***) is a international team-based 2v2 osu!Standard double-elmination tournament hosted by ![][flag_US] [Chugga444](https://osu.ppy.sh/users/12754827). This is the second installment of the Non-Professional Cup.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Staff Registration | 2020-04-12/2020-04-25 |
+| Player Registration | 2020-04-25/2020-05-14 |
+| Screening Phase | 2020-05-17/2020-05-29 |
+| Qualifiers | 2020-05-30/2020-05-31 |
+| Group Stages | 2020-06-05/2020-05-07 |
+| Round of 16 | 2020-06-12/2020-06-14 |
+| Quarterfinals | 2020-06-19/2020-06-21 |
+| Semifinals | 2020-06-26/2020-06-28 |
+| Finals | 2020-07-03/2020-07-05 |
+| Grand Finals | 2020-07-10/2020-07-12 |
+
+## Prizes
+
+| Placing | Prize |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Pending Profile Badge |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | |
+
+### Pending Badge
+
+![](images/smallbadge.png "Pending Profile Badge")
```

Comment:
```suggestion
![](images/smallbadge.png "Pending Profile Badge")
```
Also you only need to have the one with `86x40` dimension
So you can delete the upscaled @2x one.